types/craco__craco/index.d.ts

```diff
@@ -0,0 +1,341 @@
+// Type definitions for @craco/craco 6.4
+// Project: https://github.com/gsoft-inc/craco
+// Definitions by: Chuck Fan <https://github.com/fanck0605>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.6
+
+import type * as jest from '@jest/types';
+import type * as autoprefixer from 'autoprefixer';
+import type * as babel from 'babel-core';
+import type * as eslint from 'eslint';
+import type * as webpack from 'webpack';
+import type * as devServer from 'webpack-dev-server';
```

Comment:
```ts
import type { Config }  from '@jest/types';
import type { Options } from 'autoprefixer';
....
...
```
`import * ` imports everything, while the definition uses specific types only

Reply:
amended